package.json

```diff
@@ -62,24 +62,24 @@
     "browserify": "16.2.0",
     "chai": "^3.5.0",
     "cssnano": "4.1.10",
-    "dompurify": "2.0.8",
+    "dompurify": "^2.3.1",
```

Comment:
We don't have a package-lock.json, so version numbers should not be loose matches; no `^`, `~`, etc. We need to ensure that the build machine uses the same package versions we developed with and tested locally. Why are we not using package-lock.json here?